.dotnet/src/Custom/Chat/ChatMessageContent.cs

```diff
@@ -8,7 +9,7 @@ namespace OpenAI.Chat;
 public partial class ChatMessageContent : Collection<ChatMessageContentPart>
 {
     public ChatMessageContent()
-        : this([])
+        : this(Array.Empty<ChatMessageContentPart>())
```

Comment:
Was the original a bug? Are these not interchangeable? 🤔

Reply:
It's not 100% clear to me what introduced the compilation break (this type was not otherwise changed, I don't believe), but without the change I reliably get this:
```
D:\s\joseharriaga-openai-in-typespec\.dotnet\src\Custom\Chat\ChatMessageContent.cs(12,11): error CS0121: The call is ambiguous between the following methods or properties: 'ChatMessageContent.ChatMessageContent(string)' and 'ChatMessageContent.ChatMessageContent(params ChatMessageContentPart[])' [D:\s\joseharriaga-openai-in-typespec\.dotnet\src\OpenAI.csproj::TargetFramework=netstandard2.0]
```

Reply:
Yes, I was surprised... I'm curious about what exactly changed, but it didn't impact the public surface and there are too many fish being fried!